3drepo/3drepo.io
# ISSUE #4375 File format is exposed in getRevisionList endpoint

This fixes #4375

#### Description
 File format is exposed in now returned from getRevisionList endpoint
Tests and swagger documentation updated

#### Test cases
- Create a model and a revision
- Call getRevisionList enpoint
- A format property should be present in the response to show the original file's extention (eg .rvt, .ifc etc)
